administrator/components/com_guidedtours/src/Model/TourModel.php

```diff
@@ -308,4 +309,145 @@ public function getItem($pk = null)
 
         return $result;
     }
+
+    public function duplicate(&$pks)
+    {
+        $user = $this->getCurrentUser();
+        $db   = $this->getDatabase();
+
+        // Access checks.
+        if (!$user->authorise('core.create', 'com_tours') || !$user->authorise('core.create', '__guidedtour_steps')) {
+            throw new \Exception(Text::_('JERROR_CORE_CREATE_NOT_PERMITTED'));
+        }
+
+        $table = $this->getTable();
+
+        foreach ($pks as $pk) {
+            if ($table->load($pk, true)) {
+                // Reset the id to create a new record.
+                $table->id = 0;
+
+                // Alter the title.
+                $m = null;
+
+                if (preg_match('#\((\d+)\)$#', $table->title, $m)) {
+                    $table->title = preg_replace('#\(\d+\)$#', '(' . ($m[1] + 1) . ')', $table->title);
+                }
+
+                $data = $this->generateNewTitle(0, Text::_($table->title),Text::_($table->title));
+                $table->title = $data[0];
+
+                // Unpublish duplicate module
```

Comment:
copy paste error?

Reply:
May you please which error I need to change in this case?

Reply:
it is just the comment that is wrong. I think you mean "Unpublish duplicate tour" ??

Reply:
Yeah. Updating it. Thank you!